node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -724,7 +728,7 @@ cluster_name=defaultCluster
 
 # The maximum number of data points in a page, default 1024*1024
```

Comment:
This line of comment should be edited too.

Reply:
Resolved